index.html

```diff
@@ -4028,6 +4028,26 @@ <h1>Speakers</h1>
                             </div>
                         </div>
 
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                                <div class="image-holder">
+                                    <img class="background-image" alt="Tarun Kumar" src="img/tarun.jpg">
```

Comment:
Use image name as: TarunKumar.jpg or Tarun.jpg, we are following this convention throughout.